links.py

```diff
@@ -0,0 +1,51 @@
+import requests

```

Comment:
As per the task description, the result should be written in the file with JSON format. Could you please do so?

Reply:
Done

Reply:
As per the task, the output format should be the next:
```
[
  {
    "url": "https://www.does_not_exist.com/",
    "is_ok": false,
    "status_code": null
  },
  {
    "url": "http://lucumr.pocoo.org/",
    "is_ok": true,
    "status_code": 200
  },
  {
    "url": "http://jinja.pocoo.org/docs/",
    "is_ok": true,
    "status_code": 200
  }
]
```